docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
+- create number_of_floors_string, which will be appended to the details_of_system_selection string: `number_of_floors_string = ""`
+
+- get the climate zone category using the functions is_CZ_0_to_3a: `if is_CZ_0_to_3a():`
+	- set climate_zone_category to CZ_0_to_3a: `climate_zone_category = " CZ_0_to_3a"
+
+- else (the climate zone is 3b, 3c or 4 to 8): `else:`
+	- set climate_zone_category to CZ_3b_3c_or_4_to_8: `climate_zone_category = " CZ_3b_3c_or_4_to_8"
+
+- check if the building area type is residential: `if building_area_type == "RESIDENTIAL":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
```

Comment:
@KarenWGard how are we checking if the building area type is residential?
I believe this is specified at the building segment level, and the schema data element is 'heating_ventilating_air_conditioning_systems'
should this first read in the RMDI, and then for each buildingsegment, find the 'heating_ventilating_air_conditioning_systems'?

Reply:
@supriyagoel - building_area_type is passed into the function. It is calculated using the function [get_HVAC_building_area_types_and_zones](https://github.com/pnnl/ruleset-checking-tool/blob/RDS/KJW/function_get_HVAC_building_area_types_and_zones/docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md)

Reply:
makes sense. Please do list this in the section 'Function Call'

Reply:
@KarenWGard we've typically prefixed a check with the name of the function, something like get_function1.buildinhg_area_type=="residential". this clarifies that a parameter is the outcome of a function. You could see the previous RDS for the exact approach. That is really helpful to clarify when a parameter is being identified through a function

Reply:
@supriyagoel - I don't quite understand the change you're asking me to make here